spec/unit/authentication/user_authenticator/ldap_spec.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
```

Comment:
does it make sense to test in here if brute_force detector is also called when ldap is enabled ?

Reply:
:+1: